Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -454,6 +454,7 @@ public static void RegisterRoutes(IKernel kernel, RouteCollection routes)
             // Jobs
             routes.MapHttpWebJobsRoute("list-all-jobs", "", "", new { controller = "Jobs", action = "ListAllJobs" }, new { verb = new HttpMethodConstraint("GET") });
             routes.MapHttpWebJobsRoute("list-triggered-jobs", "triggered", "", new { controller = "Jobs", action = "ListTriggeredJobs" }, new { verb = new HttpMethodConstraint("GET") });
+            routes.MapHttpWebJobsRoute("list-triggered-jobs-swagger", "triggered", "/swagger", new { controller = "Jobs", action = "ListTriggeredJobsInSwaggerFormat" }, new { verb = new HttpMethodConstraint("GET") });
```

Comment:
To me, it is simply a different format or representation of the same resources.   Instead of an explicit path, could it be query string (i.e. ?format=swagger) or adhoc content-type?  
  - One can't name their job `swagger`
  - How to think the odd/even CSM Uri path

What do other people use to serve up their swaggers?